com.unity.netcode.gameobjects/Runtime/Serialization/FastBufferReader.cs

```diff
@@ -54,12 +54,12 @@ internal unsafe void CommitBitwiseReads(int amount)
 #endif
         }
 
-        private static unsafe ReaderHandle* CreateHandle(byte* buffer, int length, int offset, Allocator allocator)
+        private static unsafe ReaderHandle* CreateHandle(byte* buffer, int length, int offset, Allocator allocator, Allocator handleAllocator)
```

Comment:
so now we have `Allocator allocator` and adding `Allocator handleAllocator`. what's going on? can't someone just specify `allocator = bla` instead of `allocator = none, handleAllocator = bla`?

Reply:
When passing `allocator = Allocator.None`, it uses the buffer you pass in directly without any copies, but it still has to make an allocation for the metadata (position, length, etc). Before this change, that always used `Allocator.Temp`, but that doesn't work if you want to reference an existing buffer and keep the reader around for any length of time at all (it automatically deallocates when the stack context is left), so this change is to allow a buffer to be referenced without copying, while allocating the handle using `Allocator.Persistent` so it can last longer.

Reply:
gotcha, perhaps better parameter names and xmldoc then? :)